src/views/SelectProduct.vue

```diff
@@ -323,13 +323,24 @@ export default defineComponent({
 </script>
 
 <style scoped>
-
 .section-grid {
-  grid-template-columns: repeat(auto-fill, 200px);
+  grid-auto-flow: column;
+  grid-auto-columns: 200px;
+  grid-template-columns: none;
+  width: 100vw;
+  overflow-x: scroll;
+  scroll-snap-type: x mandatory;
+}
+
+.section-grid > div {
+  scroll-snap-align: start;
+  /* Here 20px padding-bottom is given 
+    to match the 10px bottom and 10px top margin of card */
+  padding-bottom: 20px;
 }
 
-.find {
-  padding: 0 var( --spacer-base);
+.section-grid > div > ion-card {
+  height: 100%;
```

Comment:
just an idea here that we can use a calc function instead of adding "counter padding" this will help make logic easier to understand too.

Reply:
I have used calc function instead of counter padding.